src/components/manage/BlockChooser/BlockChooserSearch.jsx

```diff
@@ -36,13 +40,13 @@ const BlockChooserSearch = ({ onChange, searchValue }) => {
         {searchValue && (
           <Button
             className="clear-search-button"
-            aria-label={intl.formatMessage(messages.search)}
+            aria-label={intl.formatMessage(messages.closeButton)}
             onClick={() => {
               onChange('');
               searchInput.current.focus();
             }}
           >
-            <Icon name={clearSVG} size="18px" />
+            <Icon className='circled' name={clearSVG} size="18px" />
```

Comment:
The use of the className here is incorrect, we don't need a circle around the icon all the time, just when focused.